client/plots/matrix.interactivity.js

```diff
@@ -131,7 +131,7 @@ export function setInteractivity(self) {
 	}
 
 	self.getImgCell = function (event) {
-		if (!self.imgBox) return null
+		if (!self.imgBox) self.imgBox = event.target.getBoundingClientRect()
```

Comment:
@congyu-lu let's make sure to check that the `event.target.tagName == 'image'` or similar:
```js
if (!self.imgBox) {
  if (event.target.tagName == 'image' && s.useCanvas) self.imgBox = event.target.getBoundingClientRect()
  else return // or throw??? 
}
```